python/tests/test_instances.py

```diff
@@ -0,0 +1,40 @@
+"""Test Ecole instance generators in Python.
+
+The instance generators are written in python.
+This file tests the instance generators with their
+default set of parameters.
+"""
+
+import ecole
+
+
+class ParameterGenerator:
+    """ Basic parameter generator to return an empty dictionary on
+        each iter.  Returning an empty dict will use the default
+        parameters of each instance generator.
+    """
+
+    def __iter__(self):
+        return self
+
+    def __next__(self):
+        return {}
+
+
+def test_Setcover(model):
+    """ Test for Setcover instance generator. """
+    parameter_generator = ParameterGenerator()
+    instances = ecole.instances.SetcoverGenerator(parameter_generator)
+    model = next(instances)
+
```

Comment:
Could we test the output more here?
Some ideas of predicates to test for could be:
 - Assert problem is minimizing
 - Assert that rhs/matrix coefficients are either 0 or 1
 - Assert variables are binary
 - Assert number of rows is that of the generator
 - Assert number of cols is that of the generator
 - Perhaps check that the problem solves without errors, probably in another test (decorated with `@pytest.mak.slow`)